lang_js/analyze/ast_js.ml

```diff
@@ -272,8 +272,11 @@ and obj_ = property list bracket
 and class_ = { 
   (* usually simply an Id *)
   c_extends: expr option;
-  c_body: property list bracket;
+  c_body: c_body_biding list bracket;
 }
+  and c_body_biding =
```

Comment:
you could add instead CEllipsis in property itself, which remove the need to introduce this intermediate c_body_biding
(which should be binding, not biding) type. It's not always clear which one is the best, but in property there is already
FieldSpread which is used to inline a variable to get all its field as in  { fld1: 1, ...var} where it's actually using '...',
so adding FieldEllipsis there make sense because it's to allow {fld1: 1, ... }

Reply:
Ok.

Reply:
Addressed